### PR TITLE
Create Code of Conduct

### PR DESCRIPTION
**Pull request by Schow94**

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# CONTRIBUTOR CODE OF CONDUCT
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+• Using welcoming and inclusive language
```
Comment (bitfield) [inline .github/CODE_OF_CONDUCT.md:10]: I think these bullets need an indent to be rendered as Markdown bullets—if you preview the file you'll see the list is collapsed into a single paragraph.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -165,4 +165,8 @@ In fact, doing a _proper_ and serious code review is a time-consuming business.
 
 Open source maintainers are just regular folk with jobs, kids, and zero free time or energy. They may not be able to drop everything and put in several hours on your PR. The task may have to wait a week or two until they can get sufficient time and peace and quiet to work on it. Don't pester them. It's fine to add a comment on the PR if you haven't heard anything for a while, asking if the reviewer's been able to look at it and whether there's anything you can do to help speed things up. Comments like 'Y U NO MERGE' are unlikely to elicit a positive response.
 
-Thanks again for helping out!
\ No newline at end of file
+Thanks again for helping out!
+
+## Code of Conduct
+
+As a contributor you can help keep the script community inclusive and open to everyone. Please read and adhere to our [Code of Conduct](CODE_OF_CONDUCT.md).
```
Comment (bitfield) [inline CONTRIBUTING.md:172]:
```suggestion
As a contributor you can help keep the `script` community inclusive and open to everyone. Please read and adhere to our [Code of Conduct](CODE_OF_CONDUCT.md).
```

**Review by bitfield — CHANGES_REQUESTED** [review]
Looks great!

File: .github/CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# CONTRIBUTOR CODE OF CONDUCT
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+* Using welcoming and inclusive language
```
Comment (Schow94) [inline .github/CODE_OF_CONDUCT.md:10]: Bullets changed to Markdown bullets

**Comment by bitfield** [comment]
Nice work @Schow94! I have a question: is there a special reason for the Code of Conduct to be in a `.github` subfolder? Does that have some special meaning to GitHub? Otherwise I would probably have put it in the root folder next to the `CONTRIBUTING` guide, for example.

**Comment by Schow94** [comment]
I saw that on another project, but I can move it to the project root folder if preferred. I don't see any reason why it needs to be in .github subfolder.

**Comment by bitfield** [comment]
Yes, I think that'll make it easier to find. [This page](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project) shows it being added to the root folder.

**Comment by bitfield** [comment]
Nice work @Schow94! Thanks a lot.

**Pull request merged**